orthoinference/README.md

```diff
@@ -100,28 +104,22 @@ Next, we want to make sure that the new `test_reactome database` can be imported
+  To verify that the graphDb has been properly populated, open `localhost:7474` (username and password default is `neo4j`), and click on the database icon at the top left. A panel titled <i>Database Information</i> should open up and display all nodes in the <a href="https://reactome.org/content/schema/">Data Model</a>. If none of this appears, chances are the neo4j instance is not using the imported graphDB. Verify the `graph.db/` folder is in fact populated. 
 
 To verify that the graphDb has been properly populated, open `localhost:7474` in your browser once the docker container is built (username and password default is <b>neo4j</b>), and click on the <i>database icon</i> at the top left. A panel titled <i>Database Information</i> should open up and display all nodes in the <a href="https://reactome.org/content/schema">Data Model</a>. If none of this appears, chances are the neo4j instance is not using the imported graphDB. Verify the `graph.db/` folder is in fact populated and make sure the location of the `graph.db/` folder is properly specified in the docker command's last argument.
 
 Once you have verified that the graphDb is populated, graph-qa can be run. Build the jar file using `mvn clean compile package` and then follow the instructions at the <a href="https://github.com/reactome/graph-qa">repo</a>. After `graph-qa` has been run, it will output the number of offending instances for each QA category, ranked by <b>urgency/priority</b>. These results can be compared with the QA results from the previous release.  QA reports can be found <a href="https://drive.google.com/drive/folders/0Byf6VJ-jol9OVnZiWjlzVlJOSGc">here</a>.
 
-5) Once you are satisfied with the results from each of these steps, send the <b>graph-qa</b> and <b>database-checker</b> results to the Curator overseeing release. <a href="https://github.com/reactome/database-checker">Database-checker</a> can be re-run fairly painlessly by following the instructions on its github page. If the Curator is satisfied with the QA results, you can move onto the next step of release. At time of writing (December 2018) this is the <i>UpdateConfig</i> step. 
+4) Once you are satisfied with the results from each of these steps, send the <b>graph-qa</b> and <b>database-checker</b> results to the Curator overseeing release. <a href="https://github.com/reactome/database-checker">Database-checker</a> can be re-run fairly painlessly by following the instructions on its github page. If the Curator is satisfied with the QA results, you can move onto one of the next steps of release. At this point in the release pipeline that could be <a href="https://github.com/reactome/data-release-pipeline/tree/develop/update-dois">UpdateDOIs</a> or <a href="https://github.com/reactome/add-links">AddLinks</a>.
```

Comment:
GitHub can be capitalized in the sentence "...following the instructions on its github page".